### PR TITLE
Changes of margins in simple pages and smooth scrolling

### PR DESCRIPTION
**Pull request by KamilWo**
Classes of container-fluid changed to container.
Removed unnecessary class.
Smooth scrolling function in custom.js.

### CODE REVIEW

File: addons/bestja_frontend_ucw/static/src/js/custom.js [hunk 1]
```diff
@@ -8,5 +8,22 @@ jQuery(window).ready(function () {
...
         }
     });    
-    /* End of function resize */    
+    /* End of function resize */
+    
+    /* Smoth scrolling */
+    
+    $(function () {
+        $('a[href*=#]:not([href=#])').click(function () {
+            if (location.pathname.replace(/^\//, '') === this.pathname.replace(/^\//, '') && location.hostname === this.hostname) {
```
Comment (QcRafal) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: przed ifem dodaj: ```$('html,body').stop();```

File: addons/bestja_frontend_ucw/static/src/js/custom.js [hunk 2]
```diff
@@ -8,5 +8,22 @@ jQuery(window).ready(function () {
...
+                var target = $(this.hash);
+                target = target.length ? target : $('[name=' + this.hash.slice(1) + ']');
+                if (target.length) {
+                    $('html,body').animate({
+                        scrollTop: target.offset().top
+                    }, 1000);
```
Comment (QcRafal) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: zamiast 1000 (1s) ustaw 500

File: addons/bestja_frontend_ucw/static/src/js/custom.js (hunk 1 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Thank you.

File: addons/bestja_frontend_ucw/static/src/js/custom.js (hunk 2 above)
Comment (KamilWo) [inline addons/bestja_frontend_ucw/static/src/js/custom.js]: Thank you.

**Comment by KrzysiekJ** [comment]
Merged in 50b7684c0f3c79bd653997f8d1bc618b2b2df9c8.

**Pull request closed without merging**